internal/controller/kustomization_configuration_error_test.go

```diff
@@ -178,7 +178,8 @@ data: {}
 	t.Run("object level workload identity feature gate enabled", func(t *testing.T) {
 		g := NewWithT(t)
 
-		t.Setenv(auth.EnvVarEnableObjectLevelWorkloadIdentity, "true")
+		auth.EnableObjectLevelWorkloadIdentity()
+		t.Cleanup(func() { auth.DisableObjectLevelWorkloadIdentity() })
```

Comment:
```suggestion
		t.Cleanup(auth.DisableObjectLevelWorkloadIdentity)
```

I forgot that here we can pass the auth func directly to t.Cleanup 😁